eclair-core/src/main/scala/fr/acinq/eclair/db/BackupHandler.scala

```diff
@@ -0,0 +1,46 @@
+package fr.acinq.eclair.db
+
+import java.io.File
+
+import akka.actor.{Actor, ActorLogging, Props}
+import fr.acinq.eclair.channel.ChannelPersisted
+
+
+/**
+  * README !
+  * This actor will synchronously make a backup of the database it was initialized with whenever it receives
+  * a ChannelPersisted event.
+  * To avoid piling up messages and entering an endless backup loop, it is supposed to be used with a bounded mailbox
+  * with a single item:
+  *
+  * backup-mailbox {
```

Comment:
Isn't there a way to enforce this configuration? That seems a bit error prone.

Overall the pattern is pretty nice.